src/composables/useNumbers.ts

```diff
@@ -82,101 +83,105 @@ export function fNum(
   );
 }
 
-export default function useNumbers() {
-  const { currency } = useUserSettings();
-  const { priceFor } = useTokens();
+export function numF(
+  number: number | string,
+  options: FNumOptions | undefined = {},
+  currency: FiatCurrency = FiatCurrency.usd
```

Comment:
Nice, I like moving it out of the composable with the currency as an option.

Reply:
Yeah, that composable has got a bit messy, but I needed to do this to make the number formatted function pure.